src/status_im/chat/models.cljs

```diff
@@ -252,23 +258,24 @@
 (fx/defn navigate-to-chat
   "Takes coeffects map and chat-id, returns effects necessary for navigation and preloading data"
   [cofx chat-id {:keys [modal? navigation-reset?]}]
-  (cond
-    modal?
-    (fx/merge cofx
-              (navigation/navigate-to-cofx :chat-modal {})
-              (preload-chat-data chat-id))
+  (when ((is-active? (get-chat cofx chat-id)))
```

Comment:
looks like there's an extra set of parentheses here

Reply:
I was about to say the same, and the second arity of `is-active?` would avoid calling `get-chat` there.

Reply:
ping @trowacat

Reply:
btw merged PR #7820 introduced function `active-chat?` an exact(apart from name) duplicate of `is-active?`

Reply:
@bitsikka yeah good point, this PR probably needs a rebase because I couldn't find active-chat? though I was pretty sure I saw it somewhere already

Reply:
Thank you all, I will look into resolving what you have pointed out and rebase tonight.